src/server/index.js

```diff
@@ -20,10 +24,12 @@ app.use("/trashInfos", trashInfosRouter);
 const trashInfoDetailsRouter = require("./routes/TrashInfoDetails");
 app.use("/trashInfoDetails", trashInfoDetailsRouter);
 
+// API for testing
 app.use(express.static("webpack-build"));
 app.get("/api/getProjectName", (req, res) =>
-  res.send({ projectName: "SmartRecycle" })
+  res.send({ projectName: "SmartCycle" })
```

Comment:
Shouldn't the project name be SmartRecycle?

Reply:
yes you are right, but this api is only for testing so I didn't update it

Reply:
It looks like it was changed though. It went from "SmartRecycle" to "SmartCycle". I would like to see it changed so everything is consistent but if it's too much work for the moment, I guess it can be left as it is.

Reply:
it is ok. I can change it if it bothers you